src/acuvity/response/helper.py

```diff
@@ -69,7 +69,7 @@ def evaluate(
 
     @staticmethod
     def _get_guard_value(
-        lookup: Dict[str, float] | None,
+        lookup: Union[Dict[str, float] , None],
```

Comment:
usually, you would use `Optional` here instead of `Union[A, None]` (which is what `Optional` stands for at the end)